README.md

```diff
@@ -4,39 +4,32 @@
 [![CircleCI](https://circleci.com/gh/Azure/aks-engine/tree/master.svg?style=svg)](https://circleci.com/gh/Azure/aks-engine/tree/master)
 [![GoDoc](https://godoc.org/github.com/Azure/aks-engine?status.svg)](https://godoc.org/github.com/Azure/aks-engine)
 
-## Overview
-
-AKS-Engine leverages ARM (Azure Resource Manager) to build Kubernetes IaaS in Azure. AKS-Engine provides convenient tooling to quickly bootstrap clusters, and implements cluster provisioning and lifecycle operations for [AKS](https://docs.microsoft.com/en-us/azure/aks), Azure's managed Kubernetes service offering.
-
-More info, including a thorough walkthrough is [here](docs/aksengine.md).
+AKS-Engine is the easiest way to provision a DIY Kubernetes cluster on Azure.
 
-Please see the [FAQ](/docs/faq.md) for answers about AKS-Engine and its progenitor ACS-Engine.
-
-## User guides
+## Overview
 
-[This guide](docs/kubernetes.md) walks you through your first cluster deployment.
+AKS-Engine provides convenient tooling to quickly bootstrap Kubernetes clusters on Azure. By leveraging [ARM (Azure Resource Manager)][ARM], AKS-Engine helps you create, destroy and maintain clusters provisioned with basic IaaS resources in Azure.
```

Comment:
This change is fine, I think we should also add something about the nature of aks-engine's relationship to AKS.

"AKS-Engine is also the library used by AKS for performing these operations to provide managed service implementations."

Reply:
done